### ISSUE TITLE
ESP8266 DS1920

### ISSUE THREAD

**Issue by Torteski** [PaulStoffregen/OneWire#26]
Hello,

There is an issue with DS1920 on the ESP8266.
When wired like:
http://www.vr-online.ru/sites/default/files/blog/2015/03/sized_3_ibutton_arduino_jpg_12791.JPG
The basic example code works with Arduino leonardo, I GET:
ROM = 10 56 25 A3 1 8 0 B1
  Chip = DS18S20
  Data = 1 34 0 4B 46 FF FF 9 10 7  CRC=7
  Temperature = 26.19 Celsius, 79.14 Fahrenheit
No more addresses.

But it does not work with ESP8266(nodeMCU 0.9) Arduino1.6.5 with ESP 2.3.0 LIB . I GET:
ROM = 10 56 25 A3 1 8 0 B1
  Chip = DS18S20
  Data = 0 FF FF FF FF FF FF FF FF FF  CRC=C9
  Temperature = -0.50 Celsius, 31.10 Fahrenheit

Always  -0.50. Any Ideas?

**Comment by PaulStoffregen** [comment]
Nope, can't help, I don't use the ESP chips.  I'm depending on the ESP8266 community to submit pull requests.
